tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -908,12 +908,9 @@ def test_OnDiskDataset_Feature_heterograph():
         )
         assert torch.equal(
             feature_data.read("edge", "author:writes:paper", "labels"),
-            torch.tensor(edge_data_label),
+            edge_data_label.clone().detach(),
         )
 
-        feature_data = None
```

Comment:
@drivanov we need to add this back since without it the windows CI will fail for some reason.
```
E                   PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\Administrator\\AppData\\Local\\Temp\\tmpstlh5ckb\\edge_writes_paper.npy'
```